src/getTime.js

```diff
@@ -18,7 +18,9 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const time = str.match(/\b[0]\d:[0-5]\d\b|[2][0-3]:[0-5]\d\b/);
```

Comment:
create a var for the pattern
```suggestion
  const matches = str.match(pattern);
```